tests/integration/run_tests.py

```diff
@@ -48,6 +57,19 @@ async def create_request_data(schema, filename):
     return data
 
 
+async def create_request_json_data(schema, filename):
+    """Create request data from pairs of schemas and filenames.
+
+    :param schema: name of the schema (folder) used for testing
+    :param filename: name of the file used for testing.
+    """
+    path_to_file = testfiles_root / schema / filename
+    path = path_to_file.as_posix()
+    with open(path, mode='r') as f:
```

Comment:
either use `aiofiles` here or change the other create_request_data function to be blocking as well

Reply:
`aiofiles` was not meant to work nicely for simple json loading ... will do my best but just saying.

Reply:
my point was just that it's weird if we use aiofiles for xml loading and standard lib for json loading :D I suggests we remove aiofiles

Reply:
i kept it, figured out what it needed.